### ISSUE TITLE
Switch Button for Smart Filters

### ISSUE THREAD

**Issue by sonnesworld** [CrocoBlock/suggestions#3082]
Hi, 

it would be great if there is a possiblity to show a switch button for some filters, e.g. here:
![image](https://user-images.githubusercontent.com/57396134/115952831-9559f280-a4e8-11eb-8570-5de7de86acdb.png)
![image](https://user-images.githubusercontent.com/57396134/115952837-a3a80e80-a4e8-11eb-9309-7c6e82c8d6f6.png)

**Comment by 79ho3ein** [comment]
Yes, it's a good idea.For example, if you can filter existing products from non-existent with this filter, it would be great.

**Comment by Lonsdale201** [comment]
Good Idea +1

**Comment by Medienor** [comment]
Please assign this custom design to radio filters and regular filters also! 

We would gladly pay Crocoblock to make this happend even faster!

1+
